src/signals/incident-management/containers/IncidentOverviewPage/components/SubNav/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Row, Column, themeSpacing, themeColor, Heading } from '@datapunt/asc-ui';
+import { Link } from 'react-router-dom';
+
+import { MAP_URL, INCIDENTS_URL } from '../../../../routes';
+
+export const Wrapper = styled(Row)`
+  margin-bottom: ${themeSpacing(5)};
+`;
+
+export const MapHeading = styled(Heading).attrs({
+  forwardedAs: 'h2',
+})`
+  margin: 0;
+  font-size: 16px;
+`;
+
+export const TabContainer = styled.div`
+  border-bottom: 2px solid ${themeColor('tint', 'level3')};
+  display: flex;
+  padding-bottom: ${themeSpacing(1)};
+
+  a {
+    text-decoration: none;
+  }
+`;
+
+export const Tab = styled.span`
+  line-height: 24px;
+  font-size: 18px;
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+  box-shadow: 0px 6px 0px 0px rgba(0, 0, 0, 0);
+  padding: 0 10px 6px;
+  color: ${themeColor('tint', 'level6')};
+
+  &.active {
+    pointer-events: none;
+    box-shadow: 0px 6px 0px 0px ${themeColor('secondary')};
+    color: ${themeColor('secondary')};
+  }
+
+  &:not(.active):hover {
+    box-shadow: 0px 6px 0px 0px rgba(0, 0, 0, 1);
+    color: ${themeColor('tint', 'level6')};
+  }
+
+  & + & {
+    margin-left: ${themeSpacing(7)};
+  }
+`;
+
+export const TabWrapper = styled(Column)`
+  justify-content: flex-end;
+  align-items: center;
+  height: ${themeSpacing(8)};
+`;
+
+const SubNav = ({ showsMap }) => (
+  <Wrapper data-testid="subNav">
+    <Column span={{ small: 1, medium: 1, big: 3, large: 6, xLarge: 6 }}>
+      {showsMap && <MapHeading>Afgelopen 24 uur</MapHeading>}
+    </Column>
+
+    <TabWrapper span={{ small: 1, medium: 1, big: 3, large: 6, xLarge: 6 }}>
+      <TabContainer>
```

Comment:
Not a fan of this construction, can we create a ticket for refactoring later on?

Reply:
Which construct?